grapher/mapCharts/MapTooltip.tsx

```diff
@@ -125,93 +130,165 @@ export class MapTooltip extends React.Component<MapTooltipProps> {
+            hidePoints: true,
+            baseFontSize: 11,
+            disableIntroAnimation: true,
+            lineStrokeWidth: 3.5,
+            annotation: {
+                entityName: this.entityName,
+                year: this.datum?.time,
+            },
+            yAxisConfig: {
+                hideAxis: false,
+                hideGridlines: false,
+                compactLabels: true,
+                // Copy min/max from top-level Grapher config if Y column == Map column
+                min: this.mapAndYColumnAreTheSame
+                    ? this.props.manager.yAxisConfig?.min
+                    : undefined,
+                max: this.mapAndYColumnAreTheSame
+                    ? this.props.manager.yAxisConfig?.max
+                    : undefined,
+                ticks: [
+                    { value: 0, priority: 1 },
+                    // Show minimum and maximum
+                    { value: -Infinity, priority: 2 },
+                    { value: Infinity, priority: 2 },
+                ],
+            },
+            xAxisConfig: {
+                hideAxis: false,
+                hideGridlines: true,
+                compactLabels: true,
+                // Always show up to the target time on the X axis,
+                // even if there is no data for it.
+                min: this.props.targetTime,
+                max: this.props.targetTime,
+                ticks: [
+                    // Show minimum and maximum
+                    { value: -Infinity, priority: 1 },
+                    { value: Infinity, priority: 1 },
+                ],
+            },
+        }
+    }
+
     render(): JSX.Element {
-        const { tooltipDatum, isEntityClickable, targetTime, manager } =
-            this.props
+        const { tooltipTarget, mapTable, datum, textColorScale } = this
+        const { isEntityClickable, targetTime, manager } = this.props
 
+        // Only LineChart and ScatterPlot allow `mapIsClickable`
         const clickToSelectMessage =
             manager.type === ChartTypeName.LineChart
                 ? "Click for change over time"
                 : "Click to select"
 
-        const { timeColumn } = this.inputTable
-        const { renderSparkBars, barColor, tooltipTarget } = this
-
+        // TODO simplify?
+        const { timeColumn } = mapTable
+        const preposition = getPreposition(timeColumn)
         const displayTime = !timeColumn.isMissing
             ? timeColumn.formatValue(targetTime)
             : targetTime
         const displayDatumTime =
-            timeColumn && tooltipDatum
-                ? timeColumn.formatValue(tooltipDatum.time)
-                : tooltipDatum?.time.toString() ?? ""
+            timeColumn && datum
+                ? timeColumn.formatValue(datum?.time)
+                : datum?.time.toString() ?? ""
+        const valueColor: string | undefined = darkenColorForHighContrastText(
+            textColorScale?.getColor(datum?.value) ?? "#333"
```

Comment:
`textColorScale` has `darkenColorForText` already, and then we're applying `darkenColorForHighContrastText` on top of that. seems a little unnecessary

Reply:
Great point – will change to use a single color scale instead, not sure what I was thinking.